assets/app.js

```diff
@@ -4,6 +4,8 @@ import './styles/app.scss';
 import 'bootstrap/js/dist/alert';
 import 'bootstrap/js/dist/collapse';
 import 'bootstrap/js/dist/dropdown';
+import 'bootstrap/js/dist/tab';
+import 'bootstrap/js/dist/util';
```

Comment:
there is no reason to import that yourselves IMO. Other Bootstrap JS files are importing the util file if they need it, and you don't use it directly by yourselves.

Reply:
You are right! I've removed the `util.js` import. Thanks.